ap/production/pileup.py

```diff
@@ -4,41 +4,58 @@
 Column production methods related to pileup weights.
 """
 
+import law
+
+from ap.production import Producer, producer
 from ap.util import maybe_import
-from ap.production import producer
 from ap.columnar_util import set_ak_column
 
 ak = maybe_import("awkward")
 
 
 @producer(
     uses={"PV.npvs"},
-    produces={"pu_weight.nominal", "pu_weight.minbias_xs_up", "pu_weight.minbias_xs_down"},
```

Comment:
This is a matter of taste of course, but I actually liked the dot naming scheme here :D Of course one could argue that the `pu_weight` isn't a field and that therefore the names should only contain underscore 🤔

Reply:
Yeah, that's true. I switched to underscore (i.e. flat and not nested under `pu_weight`) so that the alias when applying this shift is simpler. I changed it to

```python
add_aliases("minbias_xs", {"pu_weight": "pu_weight_{name}"})
```

in this PR. With dot syntax it would have been

```python
add_aliases("minbias_xs", {"pu_weight.nominal": "pu_weight.{name}"})
```

which would read as "explicitly alias to nominal". In this sense, the "new" alias is better to read and consistent with e.g. "Jet.pt" vs "Jet.pt_jec_up".